usage.txt

```diff
@@ -48,7 +48,7 @@ overwrite          if output directory for a platform already exists, replaces i
                    skipping it
 platform           all, or one or more of: darwin, linux, mas, win32 (comma-delimited if multiple).
                    Defaults to the host platform
-prune              runs `npm prune --production` on the app
+no-prune           do not run `npm prune --production` on the app
```

Comment:
Do you think this should this go after `ignore` now to keep things alphabetical?